src/Symfony/Component/Notifier/Bridge/RocketChat/Tests/RocketChatTransportTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\RocketChat\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Notifier\Bridge\RocketChat\RocketChatTransport;
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Message\ChatMessage;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class RocketChatTransportTest extends TestCase
+{
+    public function testToStringContainsProperties(): void
```

Comment:
`: void` :)

Reply:
fabbot is not enough. We need a nicobot that searches PRs for `: void` in tests. 😄

Reply:
damn :-)

Reply:
Fixed